src/Pages/Professionals/AllProfessionals/index.js

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+import { useI18n } from "providers/I18n"
+
+AllProfessionals.propTypes = {}
+
+function AllProfessionals() {
+  const { t } = useI18n()
+  return (
+    <div>
+      <h1>{t("int.all-professional-header", "professionals")}</h1>
```

Comment:
This probably displays a warning in the console about a missing i18n key int.all-professional-header. Try and fix this by adding the translation in the file `src/providers/I18n/translations.json` for both languages